src/pocketmine/event/server/RawPacketReceiveEvent.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace pocketmine\event\server;
+
+use pocketmine\event;
+use pocketmine\event\Cancellable;
+
+class RawPacketReceiveEvent extends ServerEvent implements Cancellable {
+	public static $handlerList = null;
+	private $payload;
+	private $address;
+	private $port;
+	public function __construct(string $address, int $port, string $payload) {

```

Comment:
where is the whitespace? this is all squashed together, it's unpleasant to look at.